testing/fakeextractor/fake_extractor_test.go

```diff
@@ -142,6 +142,7 @@ func TestExtract(t *testing.T) {
 	}}
 
 	type args struct {
+		//nolint:containedctx
 		ctx   context.Context
```

Comment:
note: we could remove this since all the tests use the same `context.Background()`, but then we'd not be following the `args` "pattern" which might be preferred - I think it mainly matters if we do/do not expect this to change to lot in the near future 🤷

Reply:
Don't have a strong opinion on which pattern breakage is more problematic - so sounds good to me to leave it as is and follow the args pattern instead.